stacks/openebs/deploy.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+# create namespace
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openebs
+EOF
+
+# set kubectl namespace
+kubectl config set-context --current --namespace=openebs
+
+# deploy openebs
+kubectl apply -f https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/openebs/yaml/openebs.yaml
+
+# ensure services are running
+kubectl rollout status deployment/openebs
```

Comment:
This last line fails b/c there isn't a deploy named `openebs`. The following are the deployments that are created by this stack:
```
openebs       openebs-admission-server      1/1     1            1           109s
openebs       openebs-apiserver             1/1     1            1           104s
openebs       openebs-localpv-provisioner   1/1     1            1           107s
openebs       openebs-ndm-operator          1/1     1            1           98s
openebs       openebs-provisioner           1/1     1            1           100s
openebs       openebs-snapshot-operator     1/1     1            1           99s
```

I'd advise that you create a rollout status for each component. Something like the following should work:
```sh
# ensure services are running
deployments="
  openebs-admission-server
  openebs-apiserver
  openebs-localpv-provisioner
  openebs-ndm-operator
  openebs-provisioner
  openebs-snapshot-operator
"

for deploy in $deployments; do
  kubectl rollout status deployment/"$deploy"
done
```